front-end/src/pages/EditEventDetails.jsx

```diff
@@ -73,31 +66,45 @@ export default function EditEventDetails() {
     console.log(eventDetails);
     setEvent(eventDetails);
     setTitle(eventDetails.title);
-    setDate(eventDetails.date);
-    setTime(eventDetails.time);
+    setEventStart(eventDetails.eventStart);
```

Comment:
Did you test this with the updated event serializer? the output should look like this so I'm not sure that this should work.

![image](https://github.com/crystaljobe/change-mate/assets/142848456/64555cf1-a5ca-43bd-8ada-f16afb62307a)

Reply:
It works and doesn't throw any errors, but you are correct, It won't save time or date to event details with that disparity. My goal with this was the image portion and I should have specified that. Since you brought it up ill go back and correct these issues as well!